### PR TITLE
S2 admin Structur

### PR DESCRIPTION
**Pull request by RaphaelBS-WCS**
setting up triple layout structure to prepare Admin part of the website

### CODE REVIEW

**Review by Erdnaxela45 — APPROVED** [review]

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
17 files changed??

File: src/View/component/_admin_navbar.html.twig [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<nav class="navbar navbar-expand-sm">
+    <div class="container-fluid flex-row flex-sm-column">
+        <a class="navbar-brand" href="/">
+            <h1>NATHALIE VOYANCE</h1>
```
Comment (Mcbess-san) [inline src/View/component/_admin_navbar.html.twig:4]: change h1

**Review by Mcbess-san — CHANGES_REQUESTED** [review]
check the comment

**Comment by RaphaelBS-WCS** [comment]
this pull request was replace by many others to make less files changes att a time.

**Pull request closed without merging**